This patch corrects an incompatibility problem with bison 1.50 and above.
It was taken from bonobo-activation CVS, and can be removed in the next release
of the port.

--- server/activation-context-query-parser.y.orig	Sat Oct 26 16:51:45 2002
+++ server/activation-context-query-parser.y	Sat Oct 26 16:52:00 2002
@@ -155,7 +155,7 @@
 stringlist: P_CONST_STRING { $$ = g_slist_prepend (NULL, $1); }
 	| stringlist COMMA P_CONST_STRING { $$ = g_slist_append ($1, $3); };
 
-expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); }
+expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); };
 
 expr_function: P_CONST_ID LPAREN exprlist RPAREN { $$ = qexp_function_new ($1, $3); }
 	| P_CONST_ID LPAREN RPAREN { $$ = qexp_function_new ($1, NULL); }
